packages/cli/commands/sandbox/delete.js

```diff
@@ -170,7 +173,20 @@ exports.handler = async options => {
       sandboxAccountId
     );
 
-    if (
+    if (err instanceof HubSpotAuthError) {
+      // Intercept invalid key error
+      // This command uses the parent portal PAK to delete a sandbox, so we must specify which account needs a new key
+      const regex = /\bYour personal access key is invalid\b/;
```

Comment:
This seems like an OK fix for now, but would it be possible to talk to the BE and see whether they can return the correct errors?

Reply:
Ok, interesting. Could you leave a comment here alerting anyone who changes this text that it will impact your code? I know we've got a couple of designers doing a content overview of the CLI, so we might have some changes coming up.

Reply:
^^ This is assuming there's no way to intercept the error at the point where the error message originates.

Reply:
Good call out - I would've ideally wanted to intercept by error instance AND the status code, but we're not currently passing that through... potentially another area of improvement here. I'll leave a note 👍

Reply:
Left a note here for anyone making changes coming up as well